### ISSUE TITLE
vscode eslint showing wrong error

### ISSUE THREAD

**Issue by imranbarbhuiya** [import-js/eslint-plugin-import#2354]
My vscode eslint is showing wrong error for import order and new lines. 

![image](https://user-images.githubusercontent.com/74945038/149505525-f2790ebf-6247-4c4e-ae7a-0ebed9475d4a.png)

here's the rule 
```json
{"import/order": [
      "error",
      {
        "groups": [
          "builtin",
          "index",
          "external",
          "internal",
          "sibling",
          "parent",
          "type"
        ],
        "newlines-between": "always",
        "alphabetize": { "order": "asc", "caseInsensitive": true }
      }
    ],
}
```

It's working in cli and formatting properly, but it's showing error only in vscode.

**Comment by ljharb** [comment]
This is likely because vscode’s node version doesn’t support node: builtin modules. You’ll need to update vscode’s node.

A simpler solution is to avoid the node: prefix entirely, since there’s no benefits to using it.

**Comment by imranbarbhuiya** [comment]
> You’ll need to update vscode’s node.

vscode uses different node version?

**Comment by ljharb** [comment]
Yes, it integrates its own by default.

**Comment by imranbarbhuiya** [comment]
> Yes, it integrates its own by default.

https://github.com/Microsoft/vscode/issues/18253#issuecomment-271235914

yeah i found it just now. Thank you
